crm_lead_attach_related_saleorder_report/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Crm Lead Attach Related Saleorder Report",
+    "version": "17.0.1.0.0",
+    "category": "Sale",
+    "summary": "Allow to attach sale documents into the opportunity and send them.",
+    "author": "Sygel",
+    "website": "https://github.com/sygel-technology/sy-crm",
+    "license": "AGPL-3",
+    "depends": [
+        "mail",
```

Comment:
As sale_crm depends on sale, sale depends on sales_team and sales_team depends on mail, could't you remove the dependency on main in this module?